ArticleTemplates/assets/js/modules/cards.js

```diff
@@ -1,96 +1,45 @@
 define([
-    'flipSnap',
     'modules/util'
-],
+], 
 function (
-    flipSnap,
     util
 ) {
     'use strict';
 
     var initialised = false;
+    var observer = null;
 
-    function setupGlobals() {
-        window.articleCardsInserter = articleCardsInserter;
-        window.articleCardsFailed = articleCardsFailed;
-
-        window.applyNativeFunctionCall('articleCardsInserter');
-        window.applyNativeFunctionCall('articleCardsFailed');
-    }
-
-    function articleCardsInserter(html) {
-        var relatedContentWrapper = document.getElementsByClassName('related-content__wrapper')[0];
-
-        if (relatedContentWrapper) {
-            if (!html) {
-                articleCardsFailed();
-            } else {
-                relatedContentWrapper.innerHTML = html;
-                snapToGrid();
-            }
+    function ready(config) {
+        if (!initialised) {
```

Comment:
As this only needs to run on iOS could you also add `&& GU.opts.platform === 'ios'` to this conditional?